platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/geometry/LatLngBoundsTest.java

```diff
@@ -194,6 +211,66 @@ public void testOuterUnion() {
   }
 
   @Test
+  public void northWest() {
```

Comment:
Not sure if there is a reason for it as it's valid to construct a bounds that is not found inside of the world bounds (eg. no wrapping should be enforced by LatLngBounds). The current LatLngBounds logic now matches what [core](https://github.com/mapbox/mapbox-gl-native/blob/80915992ab4e970ca005009ca1175bae24cb448b/include/mbgl/util/geo.hpp#L85) exposes. FWIW this is also the reason why this PR only validates creation of an offline region instead of enforcing this with LatLngBounds.